### PR TITLE
Fix exception handling summary

### PR DESCRIPTION
**Pull request by anmolbabu**
Signed-off-by: anmolbabu <anmolbudugutta@gmail.com>

### CODE REVIEW

**Comment by anmolbabu** [comment]
@Tendrl/tendrl-core Please review

File: tendrl/performance_monitoring/configure/configurator.py [hunk 1]
```diff
@@ -68,7 +68,6 @@ def run(self):
             while not self._complete.is_set():
                 if self._complete.is_set() or tendrl_ns.configurator_queue._closed:
                     return
-                tendrl_ns.configurator_queue._sem.acquire(True, None)
```
Comment (shtripat) [inline tendrl/performance_monitoring/configure/configurator.py]: Is it not required anymore?

File: tendrl/performance_monitoring/objects/definition/performance_monitoring.py [hunk 2]
```diff
@@ -22,7 +22,7 @@
         storage_usage:
           type: dict
         alert_count:
-          type: int
+          type: str
```
Comment (shtripat) [inline tendrl/performance_monitoring/objects/definition/performance_monitoring.py]: String?

File: tendrl/performance_monitoring/configure/configurator.py (hunk 1 above)
Comment (anmolbabu) [inline tendrl/performance_monitoring/configure/configurator.py]: I didn't see an issue without this with some fixes we did recently and the fixes here. So I think this workaround is not required anymore

File: tendrl/performance_monitoring/objects/definition/performance_monitoring.py (hunk 2 above)
Comment (anmolbabu) [inline tendrl/performance_monitoring/objects/definition/performance_monitoring.py]: Undoing it...

**Pull request merged**